<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browser</class>
 <widget class="QDialog" name="browser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>799</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListView" name="listView_type">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="tableWidget_data">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Visible</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listView_type</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>browser</receiver>
   <slot>double_click(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>856</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView_type</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>browser</receiver>
   <slot>double_click(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>885</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>browser</receiver>
   <slot>combox_change(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>774</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>821</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>browser</receiver>
   <slot>checkbox_change(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>772</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>871</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>double_click(QModelIndex)</slot>
  <slot>combox_change(QString)</slot>
  <slot>checkbox_change(bool)</slot>
 </slots>
</ui>
